README.adoc

```diff
@@ -1,214 +1,20 @@
-= Softwareentwicklung im Team: Vorbereitung auf das Projekt
-WS19/20
-:icons: font
-:icon-set: octicon
-:source-highlighter: rouge
-ifdef::env-github[]
-:tip-caption: :bulb:
-:note-caption: :information_source:
-:important-caption: :heavy_exclamation_mark:
-:caution-caption: :fire:
-:warning-caption: :warning:
-endif::[]
+= Modulhandbuch
 
-== Gruppenbildung
+== Problemstatement
 
-Für das Abschlussprojekt bilden Sie bitte eine Gruppe im GitHub Classroom, die
-aus 7 bis 9 Personen besteht und laden eine entsprechende `gruppe.yml` in AUAS
-hoch. Im Gegensatz zu den normalen Blättern haben Sie volle
-Administrationsrechte und alle Projekte sind öffentlich um den Austausch mit
-anderen Gruppen bei Integrationen zwischen Anwendungen einfacher zu gestalten.
+Das aktuelle Modulhandbuch ist ein großes Word-Dokument.
+Das komplette Dokument muss angepackt werden, um z.B ein Datum zu ändern.
+Die Darstellung der Module ist unübersichtlich.
+Es gibt keine Suchfunktion, was den oberen Punkt schwerer wiegen lässt.
+In einem Semster kann man sehen, welche Module angeboten werden, aber nicht,
+ob alle Teile des Moduls in dem Semster verfügbar sind.
 
```

Comment:
Zusätzliche Probleme aus Sicht der Verantwortlichen: Übersicht, welche Module in einem gegebenen Semester angeboten werden, liegt in einer komplett eigenen Datei.
Außerdem gibt es kein Safeguard, der verhindert, dass das Verhältnis zwischen Bachelor- und Mastermodulen nicht stimmt.